src/test/java/Homework21.java

```diff
@@ -0,0 +1,62 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.testng.annotations.Test;
+
+public class Homework21 extends BaseTest {
+
+
+    @Test
+    public static void createPlaylist() {
+        login("ksenia.strigkova@gmail.com", "te$t$tudent");
+        addPlaylist();
+        selectPlaylist();
+        enterNamePlaylist();
+        notificationText();
```

Comment:
Please,use Assertions In public static void createPlaylist()

For example  `Assert.assertTrue(notificationText());`

```
 public static boolean notificationText() {
        return wait.until(ExpectedConditions.visibilityOfElementLocated(By.cssSelector(".success"))).isDisplayed();
    }
```



Assert.assertTrue(condition): This method asserts if the condition is true or not. If not, then the exception error is thrown.

Reply:
ok thanks for your advice :)